src/model_manager/model.cpp

```diff
@@ -0,0 +1,100 @@
+#include "flockmtl/model_manager/model.hpp"
+
+namespace flockmtl {
+
+Model::Model(const nlohmann::json &model_json) {
+    // TODO: remove this when refactoring the aggregate functions
```

Comment:
Let's never have TODOs in code and add them as issues in Github

Reply:
sure I'll do that